app-server/app/src/main/scala/user/User.scala

```diff
@@ -105,35 +90,52 @@ object UserService {
     }
   }
 
-  def getUserById(id: java.util.UUID )(implicit database: Database): Future[Option[UsersRow]] = {
+  def createUser(user: UsersRowCreate)(implicit Database: Database, ec: ExecutionContext):
+      Future[UsersRow] = {
     import database.driver.api._
+    val userRow = user.toUsersRow()
 
     database.db.run {
-      Users.filter(_.id === id).result.headOption
+      Users.forceInsert(userRow)
+    } map {
+      x => userRow
     }
   }
 
-  def createUser(user: UsersRowCreate)
-    (implicit database: Database, ec: ExecutionContext): Future[UsersRow] = {
+  def createUserWithAuthId(sub: String)(implicit database: Database, ec: ExecutionContext):
+      Future[Option[UsersRow]] = {
     import database.driver.api._
-    val userRow = user.toUsersRow()
 
+    val newUUID = java.util.UUID.randomUUID
     database.db.run {
-      Users.forceInsert(userRow)
-    } map {
-      x => userRow
+      Organizations.filter(_.name === "Public").result.headOption
+    } flatMap {
+      case Some(org) => {
+        val userRow = UsersRowCreate(sub, org.id).toUsersRow()
```

Comment:
I think my comment got nixed by a different change, but just noting that it looks like `createUser` should be used here